src/tools/profile/read_profile.c

```diff
@@ -553,9 +553,8 @@ static void show_profile_data_log(profile_data_t *data, options_t *opts,
                     reqid = reqid_ctr++;
                     kh_value(&reqids, hash_it) = reqid;
                 }
-                action = "NEW ";
+                action = "NEW";
             } else {
-                assert(reqid_ctr > 1);
```

Comment:
it was added for
`UCS/DATASTRUCT, TOOLS/PROFILE: Suppress clang false-positive warnings`

Reply:
yes, static analyzer detects this problem again in Azure - will re-investigate

Reply:
@brminich it has to be moved under the handler when `kh_get()` is able to find request - `kh_value()` has to be successful in this case